public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -238,6 +238,28 @@ export function getOperationDefinitions(): QueryBuilderOperationDef[] {
       addOperationHandler: addLokiOperation,
       explainHandler: (op) => `Return log lines that contain string \`${op.params[0]}\`.`,
     },
+    {
+      id: LokiOperationId.LineContainsInsensitive,
+      name: 'Line contains (case insensitive)',
+      params: [
+        {
+          name: 'String',
+          type: 'string',
+          hideName: true,
+          placeholder: 'Text to find',
+          description: 'Find log lines that contains this text',
+          minWidth: 33,
+          runQueryOnEnter: true,
+        },
+      ],
+      defaultParams: [''],
+      alternativesKey: 'line filter',
+      category: LokiVisualQueryOperationCategory.LineFilters,
+      orderRank: LokiOperationOrder.LineFilters,
+      renderer: getLineFilterRenderer('|~', true),
+      addOperationHandler: addLokiOperation,
+      explainHandler: (op) => `Return log lines that contain string \`${op.params[0]}\`.`,
```

Comment:
I think that explainHandler here should be `explainHandler: (op) => `Return log lines that match regex \`(?i)${op.params[0]}\`.`,`